src/core/types.h

```diff
@@ -542,6 +571,7 @@ typedef struct JSFunctionBytecode {
     JSValue *cpool; /* constant pool (self pointer) */
     int cpool_count;
     int closure_var_count;
+    InlineCache *ic;
```

Comment:
why ic is defined in function but not in JSObject？

Reply:
Poly IC needs to conform to function-level locality.